warp-tls/ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## 3.3.4.1
```

Comment:
This version number doesn't seem consistent with previous numbers. There are `x.y.z.1` releases consistently follow a `x.y.z` release.

Reply:
@ysangkok `3.3.4` was the version number in `warp-tls.cabal`, so I incremented that for a bug fix. It looks like the `master` branch now has change log notes for that version, but those notes were not present when I started my branch and I failed to notice the change log had been updated on `master` when opening this PR. I'm happy to update the PR however is desired to get the right version number and notes in place.

Reply:
@Vlix Done!

Reply:
@qxjit Sorry for the __enormous__ delay, but if you could merge in `master` again and bump it to `3.4.3`, then I'll try to get one of the maintainers to make a new release :sweat_smile:

Reply:
@Vlix No worries. I've merged in `master` and bumped the version to `3.4.3` now :smile: